omnibus/cookbooks/omnibus-supermarket/metadata.rb

```diff
@@ -6,8 +6,9 @@
 long_description 'Installs/Configures supermarket in an Omnibus installation'
 version          '1.1.1'
 
+chef_version     '>= 14.0'
```

Comment:
This it the bit that could cause some confusion. The omnibus installer is supposedly bringing in Chef 13 to support the `-ctl reconfigure` action.